chart/env/prod.yaml

```diff
@@ -166,7 +166,7 @@ api:
     type: NodePort
   resources:
     requests:
-      cpu: 4
+      cpu: 200m
```

Comment:
I would go on 1

Reply:
200m may stack a lot of pods on node. Same for above and bellow

Reply:
Ok, changed.